common.blocks/input/input.post.css

```diff
@@ -1,4 +1,3 @@
-.input
-{
+:block(input) {
```

Comment:
such thing in theory is possible just in case we will process each file separately which is not possible in case of global vars (which we don't have right now but which we want to introduce in the future).